doc/reference/network_bridge.md

```diff
@@ -54,6 +54,7 @@ Key                                  | Type      | Condition             | Defau
 `bgp.peers.NAME.address`             | string    | BGP server            | -                         | Peer address (IPv4 or IPv6)
 `bgp.peers.NAME.asn`                 | integer   | BGP server            | -                         | Peer AS number
 `bgp.peers.NAME.password`            | string    | BGP server            | - (no password)           | Peer session password (optional)
+`bgp.peers.NAME.holdtime`            | string    | BGP server            | `180`                     | Peer session holdtime (optional)
```

Comment:
We should indicate in the docs that this value is in seconds.

Reply:
```suggestion
`bgp.peers.NAME.holdtime`            | string    | BGP server            | `180`                     | Peer session holdtime (in seconds; optional)
```

Reply:
If it is in seconds, why is it a string? Shouldn't it be an integer then - or if it's a string, it should accept units?

Reply:
Ops yes, sorry my fault, I'll change docs.